relentless/optimize/method.py

```diff
@@ -280,57 +323,105 @@ def max_iter(self, value):
 class SteepestDescent(Optimizer):
     r"""Steepest descent algorithm.
 
-    For an :class:`~relentless.optimize.ObjectiveFunction` :math:`f\left(\mathbf{x}\right)`,
+    For an :class:`~relentless.optimize.objective.ObjectiveFunction` :math:`f\left(\mathbf{x}\right)`,
     the steepest descent algorithm seeks to approach a minimum of the function.
 
     With an initial numerical guess for all the design variables :math:`\mathbf{x}`,
     the following iterative calculation is performed:
 
     .. math::
 
-        \mathbf{x}_{n+1} = \mathbf{x}_{n}-\alpha\nabla f\left(\mathbf{x}\right)
+        \nabla f\left(\mathbf{\hat{x}}\right) = \left[{b_1}^2 \frac{\partial f}{\partial x_1},
+                                                      \cdots,
+                                                      {b_n}^2 \frac{\partial f}{\partial x_n}\right]
 
-    where :math:`\alpha` is defined as the step size hyperparameter. The step size
-    must be specified numerically, and a :class:`LineSearch` can be performed to find
+        \mathbf{x}_{n+1} = \mathbf{x}_{n}-\alpha\nabla f\left(\mathbf{\hat{x}}\right)
```

Comment:
See my comment about rewriting the `\nabla_y`, and also make sure we document earlier up that the variables are scaled first. I would also add the second "equivalent" equation below here... basically, show the update first in scaled variables (y) then in actual variables (x) with the scaling parameters explicit.

We do need to be clear about which gradient the tolerance is defined on. Is it going to be `\nabla_x` or `\nabla_y`?